src/bosh-director/lib/bosh/director/event_log.rb

```diff
@@ -171,5 +171,9 @@ def log_entry(data = {})
         @stage.log_entry(task_entry)
       end
     end
+
+    class NullTask
```

Comment:
Could you describe a bit more why `NullTask` is added?

Is this a step for some kind of backwards compatibility? I'm struggling to see how something like `Stopper` is instantiated without the `task` argument.

Reply:
Looking at this a little bit deeper with Kevin, we found usages of `Stopper.stop` that don't pass in the `task` argument. Sounds like `NullTask` might be a stopgap until we update all these other references?